packages/block-directory/package.json

```
@@ -0,0 +1,34 @@
+{
+	"name": "@wordpress/block-directory",
+	"version": "1.0.0",
```

Nit:
```suggestion
	"version": "1.0.0-alpha.0",
```

This is the limitation of our workflow with Lerna which would try to bump to 1.0.1 during publishing to npm.